plugins/quickinspector/quickinspector.cpp

```diff
@@ -243,9 +243,9 @@ static QString qsgTextureWrapModeToString(QSGTexture::WrapMode wrapMode)
 
 static bool isGoodCandidateItem(QQuickItem *item)
 {
-    if (!item->isVisible() || item->z() < 0 || qFuzzyCompare(item->opacity() + 1.0, qreal(1.0)) ||
```

Comment:
but z < 0 doesn't mean it's invisible, so why should we exclude it? it could still be visible if all other siblings have a z even smaller

and pure quick item check also sounds bogus, as this fails with subclasses. so overall the change by christoph sounds fine to me, but the commit message needs to be improved to explain why these changes are done